### ISSUE TITLE
Ασυμβατότητα λόγω αρχείου ast.py 

### ISSUE THREAD

**Issue by ElGreKost** [slp-ntua/patrec-labs#211]
Καλησπέρα,

Το αποθετήριο περιέχει ένα αρχείο με όνομα ast.py, το οποίο προκαλεί σύγκρουση με την υπάρχουσα Python βιβλιοθήκη ast (πιθανώς εγκατεστημένη από τα requirements).

Για να λυθεί το πρόβλημα, θα πρότεινα να μετονομάσετε το αρχείο ast.py σε κάτι διαφορετικό.

Προσωρινά, το πρόβλημα επιλύεται με τη μετονομασία του αρχείου χειροκίνητα.

Ευχαριστώ!

**Comment by pxaris** [comment]
Ναι, έχεις δίκιο, υπάρχει στο standard library της Python ομώνυμο module. Μετονομάστηκε το αρχείο σε `ast_model.py`.
